.changes/unreleased/tinymce-TINY-10434-2024-01-29.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Added
+body: New `revision-history` icon
```

Comment:
We don't mention premium features in the open source changelog

Reply:
The entry is removed and svg is correctly rendered in github now